### DISCUSSION TITLE
Finetuning yourtts on a pretrained checkpoint

### DISCUSSION THREAD

**Discussion by sindhu-2507** [coqui-ai/TTS#3580] [General Q&A]
I have trained yourtts model on a 40hr male high quality english dataset for almost 1.7 million steps. Now I have two speakers male and female (20 hr each) and I have started training yourtts on top of previously trained model (40 hr male data). The alignment of male is fine and the generated voice of male is okay but the female voice is not that great. It sounds broken as if each word is pronunced independently. Is it because the previous model is completely trained on male dataset that the current model is unable to learn female voice? Is it better to train yourtts from scratch using only the 20hr male female dataset?

Thanks.

**Comment by nicemanis** [comment]
Could you share your config.json?
